src/js/components/Calendar/Calendar.js

```diff
@@ -807,11 +816,11 @@ const Calendar = forwardRef(
             : renderCalendarHeader(previousMonth, nextMonth)}
           {daysOfWeek && renderDaysOfWeek()}
           <Keyboard
-            onEnter={() =>
-              active !== undefined
-                ? selectDate(active.toISOString())
-                : undefined
-            }
+            onEnter={() => {
```

Comment:
Can we keep the old logic where we don't add an `onEnter` handler when `active` is undefined?